lib/fastlane/plugin/sentry/actions/sentry_upload_sourcemap.rb

```diff
@@ -79,9 +79,11 @@ def self.available_options
                                        description: "Distribution in release",
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :sourcemap,
-                                       description: "Path to the sourcemap to upload",
-                                       verify_block: proc do |value|
-                                         UI.user_error! "Could not find sourcemap at path '#{value}'" unless File.exist?(value)
+                                       description: "Path or an array of paths to the sourcemap(s) to upload",
```

Comment:
When we use the `Array` type, `Fastlane` will automatically turn `Strings` for us into an array. Either with one item, or split by the `,` character.

```ruby
FastlaneCore::ConfigItem.new(key: :sourcemap,
                           description: "Path or an array of paths to the sourcemap(s) to upload",
                           type: Array,
                           verify_block: proc do |values|
                             values.each do |v|
                               UI.user_error! "Could not find sourcemap at path '#{v}'" unless File.exist?(v)
                             end
                           end),
```

Reply:
🙇 Thanks for the tip! This was indeed much simpler. Updated.